sse2neon.h

```diff
@@ -419,6 +419,39 @@ FORCE_INLINE float _mm_cvtss_f32(__m128 a)
     return vgetq_lane_f32(vreinterpretq_f32_m128(a), 0);
 }
 
+// Convert the lower single-precision (32-bit) floating-point element in a to a
+// 32-bit integer, and store the result in dst.
+//
+//   dst[31:0] := Convert_FP32_To_Int32(a[31:0])
+//
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_cvtss_si32
+#define _mm_cvtss_si32(a) _mm_cvt_ss2si(a)
+
+// Convert the lower single-precision (32-bit) floating-point element in a to a
+// 64-bit integer, and store the result in dst.
+//
+//   dst[63:0] := Convert_FP32_To_Int64(a[31:0])
+//
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_cvtss_si64
+FORCE_INLINE int _mm_cvtss_si64(__m128 a)
+{
+#if defined(__aarch64__)
+    return vgetq_lane_s64(
+        vreinterpretq_s64_s32(vcvtnq_s32_f32(vreinterpretq_f32_m128(a))), 0);
+#else
+    float32_t data = vgetq_lane_f32(vreinterpretq_f32_m128(a), 0);
+    float32_t diff = data - floor(data);
```

Comment:
It is feasible to replace the following code snip with round implementations:
* `__builtin_roundf`
* `roundf`

Then, use `ROUND_IMPL(vgetq_lane_f32(vreinterpretq_f32_m128(a), 0))`.

Reply:
We can not replace it since the `roundf` and `__buildin_roundf` use the rounding mode `round to nearest, where ties round away from zero`.
The x86 intrinsics use the rounding mode `round to nearest, where ties round to the nearest even digit in the required position`.